aspose/src/main/java/org/frankframework/extensions/aspose/services/conv/impl/convertors/InvalidPdfException.java

```diff
@@ -0,0 +1,22 @@
+/*
+   Copyright 2025 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.extensions.aspose.services.conv.impl.convertors;
+
+public class InvalidPdfException extends Exception {
```

Comment:
Kan dit een IbisException worden? :)
```suggestion
public class InvalidPdfException extends IbisException {
```